docker/Dockerfile

```diff
@@ -1,19 +1,31 @@
 FROM openjdk:8-jre-alpine
 
-ENV APP_PORT=8083
-ENV GB_BACKEND_VERSION=0.4.0-RC1
-ENV GB_BACKEND_WAR_URL=https://repo.thehyve.nl/content/repositories/releases/nl/thehyve/gb-backend/${GB_BACKEND_VERSION}/gb-backend-${GB_BACKEND_VERSION}.war
+ENV APP_PORT 8083
+ENV GB_BACKEND_WAR_PATH https://repo.thehyve.nl/content/repositories/releases/nl/thehyve/gb-backend
 
 # non-overridable environment variables
-ENV APP_CONFIG_DIR=gb-backend.config.yml
+ENV APP_USR gb_backend_usr
+ENV APP_USR_HOME "/home/${APP_USR}"
+ENV APP_CONFIG_DIR ${APP_USR_HOME}/gb-backend.config.yml
 
-COPY entrypoint.sh /entrypoint.sh
+COPY docker/entrypoint.sh /entrypoint.sh
+COPY gb-backend/VERSION /VERSION
```

Comment:
@gijskant Should it use the version file from gb-backend or should there be a separate file for docker? Or better to keep using the variable (previous way) for now?